src/lib/stitching/gravity/schema.ts

```diff
@@ -8,7 +8,12 @@ import {
 } from "graphql-tools"
 import { readFileSync } from "fs"
 
-const rootFieldsAllowList = ["agreement", "viewingRoom", "viewingRooms"]
+const rootFieldsAllowList = [
+  "agreement",
+  "viewingRoom",
+  "viewingRooms",
+  "viewingRoomsConnection",
```

Comment:
Strange that the connection was added (and other one deprecated) - but we were never using it in the schema! (although I see we were using it via `_unused`)